src/parser/rogue/outlaw/modules/spells/BetweenTheEyes.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+import Analyzer from 'parser/core/Analyzer';
+
+import DamageTracker from 'parser/shared/modules/AbilityTracker';
+
+import BetweenTheEyesDamageTracker from './BetweenTheEyesDamageTracker';
+
+class BetweenTheEyes extends Analyzer {
+  static dependencies = {
+    damageTracker: DamageTracker,
+    betweenTheEyesDamageTracker: BetweenTheEyesDamageTracker,
+  };
+
+  constructor(...args) {
+    super(...args);
+    this.active = !this.selectedCombatant.hasTalent(SPELLS.SLICE_AND_DICE_TALENT.id) && !(this.selectedCombatant.hasTrait(SPELLS.ACE_UP_YOUR_SLEEVE.id) || this.selectedCombatant.hasTrait(SPELLS.DEADSHOT.id));
+  }
+
+  get thresholds() {
+    const total = this.damageTracker.getAbility(SPELLS.BETWEEN_THE_EYES.id);
+    const filtered = this.betweenTheEyesDamageTracker.getAbility(SPELLS.BETWEEN_THE_EYES.id);
+
+    // the betweenTheEyesDamageTracker is tracking casts when you should be using BTE, so subtract from the total to get all the casts when you shouldn't
+    const incorrectCasts = total.casts - filtered.casts;
+
+    return {
+      actual: incorrectCasts / total.casts,
+      isGreaterThan: {
+        minor: 0,
+        average: 0.05,
+        major: 0.1,
+      },
+      style: 'number',
+    };
+  }
+
+  suggestions(when) {
+    when(this.thresholds).addSuggestion((suggest, actual, recommended) => {
+      return suggest(<>You casted <SpellLink id={SPELLS.BETWEEN_THE_EYES.id} /> without having <SpellLink id={SPELLS.RUTHLESS_PRECISION.id} /> active. When you don't have either the <SpellLink id={SPELLS.ACE_UP_YOUR_SLEEVE.id} /> or <SpellLink id={SPELLS.DEADSHOT.id} /> traits, the only time you should use <SpellLink id={SPELLS.BETWEEN_THE_EYES.id} /> as your damaging finisher is during <SpellLink id={SPELLS.RUTHLESS_PRECISION.id} />.</>)
```

Comment:
Something I couldn't quite decide on, is the second sentence here needed? Is it preferable to give all of the information about why something should/shouldn't be done, or stick to only what's relevant to that player in that fight?

Reply:
In suggestions it's good to give more information